_tools/release_docs/api-reference-process.go

```diff
@@ -67,7 +67,7 @@ func stripHTML(filename string) error {
 // replace http://releases.k8s.io/HEAD/docs/xxx.md with
 // http://kubernetes.io/v1.0/docs/xxx.html
 func fixLinks(fileName, outputDir string) error {
-	mdRE := regexp.MustCompile(`http://releases.k8s.io/HEAD/docs(.*?\.)md`)
```

Comment:
Should we only replace http://releases.k8s.io/v1.0/docs/xxx.md with http://kubernetes.io/v1.0/docs/xxx.html?
Its versionize-docs's job to convert http://releases.k8s.io/HEAD/docs/xxx.md to http://releases.k8s.io/v1.0/docs/xxx.md, right?